pkg/ccl/backupccl/backupbase/constants.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 The Cockroach Authors.
```

Comment:
Since `backupccl` is in the path, the prefix `backup` in `backupbase` seems redundant? Maybe just name the packages `base`, `destination`, and `utils`?

Reply:
hmmm so when these packages are used by others it'll be `backupbase.foo` which might be more informative than just `base.foo`? I agree the stutter is not ideal but I think we have some precedence here:

`changfeedccl/changefeedbase`, `kv/kvserver/kvbase` being a few of them.

Reply:
The schemachanger package settled on a sort abbreviation for their "sub packages" (`scexec`, `scplan`), but I'm not sure `bk` or `bu` have the same ring to them.